src/plugins/kibana_react/public/page_template/no_data_page/no_data_config/no_data_config_page.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { EuiPageTemplate } from '@elastic/eui';
+import React from 'react';
+import classNames from 'classnames';
+import { NO_DATA_PAGE_TEMPLATE_PROPS, NoDataPage } from '../no_data_page';
+import { withSolutionNavbar } from '../../with_solution_navbar';
+import { KibanaPageTemplateProps } from '../../page_template';
+
+export const NoDataConfigPage = (props: KibanaPageTemplateProps) => {
+  const { className, noDataConfig } = props;
+
+  if (!noDataConfig) {
+    return null;
+  }
+
+  const template = NO_DATA_PAGE_TEMPLATE_PROPS.template;
+
+  const classes = classNames(
+    'kbnPageTemplate',
+    { [`kbnPageTemplate--${template}`]: template },
+    className
+  );
```

Comment:
These are duplicated here and in the normal template, since we want them to always stay the same and in-sync, I'd suggest moving it back to the `page_template.tsx` file and pass through.